lab_4_fill_words_by_ngrams/main.py

```diff
@@ -56,6 +77,16 @@ def _postprocess_decoded_text(self, decoded_corpus: tuple[str, ...]) -> str:  #
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not isinstance(decoded_corpus, tuple) or not decoded_corpus:
+            raise ValueError
+        decoded_corpus = ' '.join(decoded_corpus).replace(f' {self._end_of_word_token}', '.')
```

Comment:
`f' {self._end_of_word_token}'` to a variable, please

Reply:
fixed